### PR TITLE
Removed Accounts.html

### LINKED ISSUE #7796: Remove 2018 firefox/accounts page
## Description

I can't find any locales that are active but have not translated the 2019 page. Which means the 2018 page `accounts.html` is not being displayed to anyone. We should remove the old page.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Old accounts page is retired

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] Am I wrong about the l10n activation status?

#### Tasks
- [ ] Remove `accounts.html` and associated CSS, JS, and assets.
- [ ] Update the `firefox_accounts` view to remove references to it
- [ ] Update tests

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #7796

### CODE REVIEW

File: bedrock/base/templates/macros.html [hunk 1]
```diff
@@ -144,7 +144,7 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
   </section>
 {%- endmacro %}
 
-{# Docs: https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#linking-to-accountsfirefoxcom #}
+{# Docs: https://bedrock.readthedocs.io/en/latest/firefox-accounts-2019.html#linking-to-accountsfirefoxcom #}
```
Comment (alexgibson) [inline bedrock/base/templates/macros.html]: The changes in this file are not related to this issue, there's no need to update these links.

File: bedrock/firefox/templates/firefox/accounts.html [hunk 2]
```diff
@@ -1,346 +0,0 @@
...
-{% block page_desc %}
-{% if l10n_has_tag('firefox_send_20190420') %}
-  {{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One login – Power and privacy everywhere.') }}
-{% else %}
-  {{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.') }}
-{% endif %}
-{% endblock %}
-
-{% block page_css %}
- {{ css_bundle('firefox_accounts') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/accounts.html]: Please remove this bundle in [static-bundles.json](https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L767-L773)

You can also delete the `css/firefox/accounts.scss` file.

File: bedrock/firefox/templates/firefox/accounts.html [hunk 3]
```diff
@@ -1,346 +0,0 @@
...
-        </div>
-        {# L10n: Line break below for visual formatting only. #}
-        <div class="show-fxa-android">
-          <a class="mzp-c-button mzp-t-secondary" rel="external" href="https://support.mozilla.org/kb/sync-bookmarks-tabs-history-and-passwords-android">{{ _('Set up Sync on your <br>Android device') }}</a>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="mzp-c-hero-image">
-    {{ high_res_img('firefox/accounts/fxa-hero-img.png', {'alt': '', 'width': '1200', 'height': '770'}) }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/accounts.html]: I believe this (and some of the other images referenced in this template) are no longer needed and can be removed. Please check carefully through the images before removing them.

https://github.com/mozilla/bedrock/tree/master/media/img/firefox/accounts

File: bedrock/firefox/tests/test_views.py [hunk 4]
```diff
@@ -998,7 +998,7 @@ def test_accounts_page_legacy(self, render_mock):
         req = RequestFactory().get('/firefox/accounts/')
         req.locale = 'fr'
         views.firefox_accounts(req)
-        render_mock.assert_called_once_with(req, 'firefox/accounts.html', ANY)
+        render_mock.assert_called_once_with(req, 'firefox/accounts-2019.html', ANY)
```
Comment (alexgibson) [inline bedrock/firefox/tests/test_views.py]: I don't think this particular test (`test_accounts_page_legacy`) is needed any longer. You can remove this test, as opposed to update the template name here.

File: bedrock/firefox/views.py [hunk 5]
```diff
@@ -930,7 +930,7 @@ def firefox_accounts(request):
     if lang_file_is_active('firefox/accounts-2019', locale):
         template_name = 'firefox/accounts-2019.html'
     else:
-        template_name = 'firefox/accounts.html'
+        template_name = 'firefox/accounts-2019.html'
```
Comment (alexgibson) [inline bedrock/firefox/views.py]: You can remove the whole `if else` conditional here, and just have:

```
template_name = 'firefox/accounts-2019.html
```

File: bedrock/firefox/templates/firefox/accounts.html [hunk 6]
```diff
@@ -1,346 +0,0 @@
...
-        </div>
-      </div>
-    </div>
-
-  </div>
-</section>
-{% endblock %}
-
-{% block js %}
-  {{ js_bundle('firefox_accounts') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/accounts.html]: You can remove this bundle: https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L1386-L1396

The `js/firefox/accounts.js` file can also be deleted.

**Comment by alexgibson** [comment]
This looks good, thanks @Patil2099 - just a few more things to tidy up.

**Comment by Patil2099** [comment]
@alexgibson Can you please take a look at the changes that I made?

File: bedrock/firefox/templates/firefox/accounts.html (hunk 6 above)
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/accounts.html]: You still need to delete this bundle from `static-bundles.json`

**Comment by Patil2099** [comment]
@alexgibson  Is this correct now?

**Comment by alexgibson** [comment]
@Patil2099 The images in /firefox/accounts/trailhead/ are still in use in this latest version of the page (they are now 404). Please test your changes, thanks!

**Comment by Patil2099** [comment]
> The last commit removed a directory of images that are still in use.

Is this correct now? @alexgibson 
Thanks!

File: bedrock/firefox/tests/test_views.py [hunk 7]
```diff
@@ -992,13 +992,6 @@ def test_accounts_page_2019(self, render_mock):
         views.firefox_accounts(req)
         render_mock.assert_called_once_with(req, 'firefox/accounts-2019.html', ANY)
 
-    @patch('bedrock.firefox.views.l10n_utils.render')
-    @patch.object(views, 'lang_file_is_active', lambda *x: False)
-    def test_accounts_page_legacy(self, render_mock):
-        req = RequestFactory().get('/firefox/accounts/')
-        req.locale = 'fr'
-        views.firefox_accounts(req)
-        render_mock.assert_called_once_with(req, 'firefox/accounts.html', ANY)
```
Comment (alexgibson) [inline bedrock/firefox/tests/test_views.py]: Oh, it looks like there's one last flake 8 warning here:

```
./bedrock/firefox/tests/test_views.py:997:1: E303 too many blank lines (3)
1     E303 too many blank lines (3)
```

I believe it wants each test to be separated by 2 blank lines, but you have 3 here. Deleting this line should fix it, then we're good to merge.

**Comment by Patil2099** [comment]
@alexgibson I think everything is good now!
Thanks For the Help

**Comment by alexgibson** [comment]
Rebased and merged to master in https://github.com/mozilla/bedrock/commit/74f0ab68e8d53cd6b5504254b830a62fa75067e5

Thanks for all the hard work on this one @Patil2099 🏅
